tests/ballerina-integration-test/src/test/resources/grpc/clients/grpc_secured_unary_client.bal

```diff
@@ -31,7 +31,7 @@ function testUnarySecuredBlocking() returns (string) {
                 name: "TLSv1.2",
                 versions: ["TLSv1.2","TLSv1.1"]
             },
-            ciphers:["TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA"],
+            ciphers:["TLS_RSA_WITH_AES_128_CBC_SHA"],
```

Comment:
Just for my understanding, why we had to switch from Diffie Hellman to RSA?

Reply:
Upgraded openssl version does not support that cipher.